pkg/proxy/winkernel/hnsV2.go

```diff
@@ -89,18 +91,31 @@ func (hns hnsV2) getEndpointByIpAddress(ip string, networkName string) (*endpoin
 			equal = endpoint.IpConfigurations[0].IpAddress == ip
 		}
 		if equal && strings.EqualFold(endpoint.HostComputeNetwork, hnsnetwork.Id) {
+			pol := transformToPoliciesinfo(endpoint.Policies)
 			return &endpointsInfo{
 				ip:         endpoint.IpConfigurations[0].IpAddress,
 				isLocal:    uint32(endpoint.Flags&hcn.EndpointFlagsRemoteEndpoint) == 0, //TODO: Change isLocal to isRemote
 				macAddress: endpoint.MacAddress,
 				hnsID:      endpoint.Id,
 				hns:        hns,
+				policies:   pol,
 			}, nil
 		}
 	}
 
 	return nil, fmt.Errorf("Endpoint %v not found on network %s", ip, networkName)
 }
+func transformToPoliciesinfo(hcnendpointpolicies []hcn.EndpointPolicy) []*policiesinfo {
+	var endpointPolicies []*policiesinfo
+	for _, po := range hcnendpointpolicies {
+		var policy policiesinfo
```

Comment:
Include a log statement here for po

Reply:
There is one for policy on line 115 ... but not sure if you were suggesting that we should have a log for both structs?

Reply:
Yes we should log po (each policy we inspect) as well as the policyInfo we create